src/custom-elements/webapp-events.js

```diff
@@ -2,11 +2,25 @@ import {
   addMessagingEventListener,
   dispatchEvent,
   EVENT_SETTINGS_CHANGED,
+  EVENT_URL_CHANGED,
   READY_EVENT,
   RESPONSE_PACKAGE_INFO_EVENT,
 } from '../events-shared.js';
 import * as store from './store.js';
 
+const listenForUrlChange = () => {
```

Comment:
I think I found a better solution for this issue - sending a message from `background.js` about URL change using `browser.tabs.onUpdated` event.